tinybytes/src/lib.rs

```diff
@@ -270,6 +273,16 @@ impl fmt::Debug for Bytes {
     }
 }
 
+#[cfg(feature = "serde")]
+impl Serialize for Bytes {
```

Comment:
I know it's covered in trace-utils, but Is this something we should explicitly test in tinybytes as well?